services/apps/premium/members_enrichment_worker/src/sources/serp/service.ts

```diff
@@ -0,0 +1,209 @@
+import axios from 'axios'
+
+import { Logger, LoggerBase } from '@crowd/logging'
+import { MemberEnrichmentSource, MemberIdentityType, PlatformType } from '@crowd/types'
+
+import {
+  IEnrichmentService,
+  IEnrichmentSourceInput,
+  IMemberEnrichmentDataNormalized,
+} from '../../types'
+
+import { IMemberEnrichmentDataSerp, IMemberEnrichmentSerpApiResponse } from './types'
+
+export default class EnrichmentServiceSerpApi extends LoggerBase implements IEnrichmentService {
+  public source: MemberEnrichmentSource = MemberEnrichmentSource.SERP
+  public platform = `enrichment-${this.source}`
+
+  public enrichableBySql = `
+  (members."displayName" like '% %') AND 
```

Comment:
What's the idea? People with a space in their name?

Reply:
yeah, people with multiple-word names. Single-word names are not good enough when searching in google.

But now I'm thinking, since we're already using other stuff to distinguish (like website, location w/e), maybe we can remove this

Reply:
the test results with one-word display names aren't great.
- With multiple-word names, the correctness using GitHub identities in queries was %92.4, when using single-word names it dropped to %65 
- With multiple word names, queries with the website was %100, when using single-word names it dropped to %75
I'm keeping this for now, since it's more important to have more correct answers than more enrichable members